package.json

```diff
@@ -42,6 +42,13 @@
   "pnpm": {
     "patchedDependencies": {
       "@jsdevtools/ono@7.1.3": "patches/@jsdevtools__ono@7.1.3.patch"
+    },
+    "overrides": {
+      "word-wrap": "^1.2.4",
+      "semver": "^7.5.4",
+      "@types/react": "18.0.27",
+      "goober": "2.1.9",
+      "react-hot-toast": "2.2.0"
```

Comment:
Can you remove the `resolutions` field above which basically duplicates this?

Reply:
Done